include/BSMPT/transition_tracer/transition_tracer.h

```diff
@@ -181,14 +195,24 @@ class TransitionTracer
 
   /**
    * @brief Store the list of bounce solutions
-   *
    */
   std::vector<BounceSolution> ListBounceSolution;
 
   /**
    * @brief output data storage
    */
   output output_store;
+
+  /**
+   * @brief CheckMassRatio
```

Comment:
Can you add a description of the function?

Reply:
[Done.](https://github.com/phbasler/BSMPT/pull/180/commits/ecfc32385f01cf1b0441c08f1bf50ffb81615dd4)